driver-core/src/main/java/com/datastax/driver/core/SimpleStatement.java

```diff
@@ -193,6 +194,19 @@ public Object getObject(int i) {
     }
 
     /**
+     * Returns the named values as a {@code Map<String, Object>}.
+     *
+     * @return the named values of this statement.
+     * @throws IllegalStateException     if this statement does not have named values.
+     */
+    public Map<String, Object> getObjectMap() {
```

Comment:
Sorry if I'm arriving a bit late for the party (since #610 is already merged).

1. Couldn't we rename this method to `getNamedValues()`?
2. Since we're already augmenting the API, and for the sake of symmetry, couldn't we also provide a `Object[] getValues()` method for positional parameters? I recall users asking for this frequently, precisely for debugging purposes.
3. While at it, since we have `Object getObject(int)` can't we have a symmetric  `Object getObject(String)`?

I know all of this should have been addressed in JAVA-1037, but it sounds like a good opportunity here to add those missing methods.